app/app.py

```diff
@@ -32,7 +33,18 @@ def parse_flow(args):
     elif args.get("flow") == "destination":
         return "destination"
     else:
-        raise exceptions.ServerError("Unsupported flow specified", status_code=500)
+        raise exceptions.ServerError("Unsupported flow specified. Must be either origin (default) or destination.", status_code=500)
+
+
+def parse_mode(args):
```

Comment:
this might be extra cautious, but maybe we run a lowercase method `.lower()` on the mode param string before the if elses to normalize. Esp if we are thinking about other clients/API users.

Reply:
👍 agreed.